src/native/libs/System.Native/pal_datetime_time_zone_data.c

```diff
@@ -0,0 +1,81 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#include <stdint.h>
+#include <string.h>
+#include <assert.h>
+#include <stdbool.h>
+#include <minipal/utils.h>
+
+#ifdef TZ_DATA_ENABLED
+#include <pal_datetime_time_zone_data-config.h>
+
+#define STR_IMPL(...) #__VA_ARGS__
+#define STR(...) STR_IMPL(__VA_ARGS__)
+
+__asm(
```

Comment:
is a comment about this inline asm useful?

Reply:
I've added one, let me know if it helps.

In general, the code is straightforward, but it's muddled by the usual obscurity the syntax. E. g. `\@` - "generate unique integer for each macro expansion", `.` - "current position (in the current section)".